backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/ir/util/IrUnboundSymbolReplacer.kt

```diff
@@ -19,36 +19,6 @@ import org.jetbrains.kotlin.ir.symbols.*
 import org.jetbrains.kotlin.ir.types.toKotlinType
 import org.jetbrains.kotlin.ir.visitors.*
 
-@Deprecated("")
-internal fun IrModuleFragment.replaceUnboundSymbols(context: Context) {
```

Comment:
@alexander-gorshenev JS team remove such function and there is no call for it now, so also remove.